modules/siddhi-core/src/main/java/org/wso2/siddhi/core/stream/output/sink/SinkMapper.java

```diff
@@ -38,25 +43,32 @@
     private String type;
     private SinkListener sinkListener;
     private OptionHolder optionHolder;
-    private TemplateBuilder payloadTemplateBuilder = null;
+    private Map<String, TemplateBuilder> templateBuilderMap = null;
```

Comment:
What's the key of the map? Have a comment to describe the key and the value.

Reply:
Key is specified in sink mapper's paylod as @payload(key="template"). Value is the template builder for the given template.